src/bmark/bmark_solvers.jl

```diff
@@ -1,4 +1,7 @@
-export bmark_solvers, profile_solvers, bmark_and_profile
+export bmark_solvers, profile_solvers, profile_solvers_by_time,
+       profile_solvers_by_iterations, profile_solvers_by_evaluations,
+       bmark_and_profile, bmark_and_profile_by_time,
+       bmark_and_profile_by_iterations, bmark_and_profile_by_evaluations
```

Comment:
I see, to do it only once instead of one for each cost.
This would work differently for the custom cost, however.

I'm thinking
- `:eval` and `:time` as pre-determined cost metrics;
- keyword arguments `custom_costs :: Dict{Symbol,Function}` for other possible metrics;
- Create a profile for each cost metric, returning `profiles :: Dict{Symbol,BMProf}`, where the keys are the symbols `:eval`, `:time` and the custom ones.